### PR TITLE
chore(buildkite): move documentation site check to zeit

### PR DESCRIPTION
**Pull request by gergelyke**

### CODE REVIEW

File: scripts/link-checker.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+set -e
+
+branchUrl=$(echo $BUILDKITE_BRANCH | tr / -)
+url="https://baseui-git-$branchUrl.uber-ui-platform.now.sh/"
+
+attempt_counter=0
+max_attempts=20
+
+until $(curl --output /dev/null --silent --head --fail $url); do
```
Comment (chasestarr) [inline scripts/link-checker.sh]: what is this protecting against? attempting to reach the base-url before it's ready? I've seen the link checker fail on some thrid-party links while the `blc` command is running. Mostly interested to learn why this was required

Comment (gergelyke) [reply scripts/link-checker.sh]: yeah - this part just makes sure that the zeit deployment is ready

Comment (chasestarr) [reply scripts/link-checker.sh]: ok, so the key difference is that we don't run the site locally in CI

Comment (gergelyke) [reply scripts/link-checker.sh]: yes, that's correct

**Pull request merged**